examples/gno.land/r/demo/games/shifumi/shifumi.gno

```diff
@@ -0,0 +1,100 @@
+package shifumi
+
+import (
+	"errors"
+	"std"
+	"strconv"
+)
+
+const (
+	empty = iota
+	rock
+	paper
+	scissors
+	last
+)
+
+type game struct {
+	player1, player2 std.Address // shifumi is a 2 players game
+	move1, move2     int         // can be empty, rock, paper, or scissors
+}
+
+var games = []*game{}
```

Comment:
avl.Tree for scalability? can use `seqid` to have a compatible index

then the Render() page could render the, say, last 100 games.

Reply:
using now `avl.Tree` and `seqid`